src/main/java/uk/gov/hmcts/probate/controller/TaskListController.java

```diff
@@ -16,11 +18,25 @@
 @RequiredArgsConstructor
 @RequestMapping("/tasklist")
 public class TaskListController {
+    private final ObjectMapper objectMapper;
 
     private final CallbackResponseTransformer callbackResponseTransformer;
 
     @PostMapping(path = "/update")
     public ResponseEntity<CallbackResponse> update(@RequestBody CallbackRequest request) {
+        logRequest("/tasklist/update", request);
         return ResponseEntity.ok(callbackResponseTransformer.updateTaskList(request));
     }
+
+    private void logRequest(String uri, CallbackRequest callbackRequest) {
```

Comment:
we can probably get rid of all of this - its just for logging, and we shouldnt be pushing all request info into logs